src/prop/theory_proxy.h

```diff
@@ -118,6 +118,14 @@ class TheoryProxy : public Registrar
   theory::TrustNode removeItes(TNode node,
                                std::vector<theory::TrustNode>& newLemmas,
                                std::vector<Node>& newSkolems);
+  /**
+   * Get the skolems within node and their corresponding definitions, store
+   * them in skAsserts and sks respectively. Note that this method does not
+   * traverse to include skolems that occur in skAsserts.
```

Comment:
This is a bit confusing. I imagine that `skAsserts` are not in `node`, so one would not expect them to be traversed anyway. The disclaimer might be accompanied of this clarification.